### PR TITLE
[gli] upgrade the gli to latest

### PR DESCRIPTION
**Pull request by jixingcn**
Just upgrade the package `gli`

- #### What does your PR fix?  
  Just update the package `gli` to the least.

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  Just update the commit, sha512, patch and version.

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  Yes.

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  Yes.

### CODE REVIEW

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

**Comment by JonLiu1993** [comment]
@jixingcn ，Thanks for your pr,after finish your suggesstion changes,please run command "./vcpkg x-add-version gli --overwrite-version" then commit the changes again

File: ports/gli/disable-test.patch [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
++++ b/CMakeLists.txt
+@@ -72,7 +72,7 @@ endmacro(addExternalPackageGTC)
+ # Add subdirectory
+ 
+ add_subdirectory(gli)
+-add_subdirectory(test)
++#add_subdirectory(test)
+ #add_subdirectory(doc)
+ 
+ ################################
```
Comment (JonLiu1993) [inline ports/gli/disable-test.patch:13]: Could you please keep the original format of EOL for this patch, maybe LF?

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

**Comment by JonLiu1993** [comment]
@jixingcn ，Thanks for your pr,after finish your suggesstion changes,please run command "./vcpkg x-add-version gli --overwrite-version" then commit the changes again

File: ports/gli/disable-test.patch (hunk 1 above)
Comment (jixingcn) [reply ports/gli/disable-test.patch:13]: Ok

Comment (jixingcn) [reply ports/gli/disable-test.patch:13]: I check that and its EOF is LF. What do I make mistake?

**Comment by jixingcn** [comment]
> @jixingcn ，Thanks for your pr,after finish your suggesstion changes,please run command "./vcpkg x-add-version gli --overwrite-version" then commit the changes again

I just commit it again. Thanks for your check.

File: ports/gli/disable-test.patch (hunk 1 above)
Comment (JonLiu1993) [reply ports/gli/disable-test.patch:13]: Now the format of LF is different from before, so we should be changed to CRLF

**Review by JonLiu1993 — APPROVED** [review]

**Comment by JonLiu1993** [comment]
Thanks for your pr.

**Comment by jixingcn** [comment]
> Thanks for your pr.

👌

**Review by ras0219-msft — APPROVED** [review]

**Review by jixingcn — COMMENTED** [review]
ok

**Comment by ras0219-msft** [comment]
Thanks for the PR!

**Pull request merged**